packages/ui/src/components/library/Autocomplete.tsx

```diff
@@ -96,42 +92,19 @@ const Autocomplete = <
         renderOption={renderOption}
         popupIcon={<HiOutlineChevronDown />}
         renderInput={renderInput}
+        PopperComponent={PopperMy}
```

Comment:
We can pass PopperStyled directly without the function wrapper 
```
PopperComponent={PopperStyled} 
```

Reply:
working nicely :+1: